package-lock.json

```diff
@@ -2327,7 +2327,7 @@
                 },
                 "load-json-file": {
                     "version": "2.0.0",
-                    "resolved": "https://registry.npmjs.org/load-json-file/-/load-json-file-2.0.0.tgz",
+                    "resolved": "http://registry.npmjs.org/load-json-file/-/load-json-file-2.0.0.tgz",
```

Comment:
Thats a bit strange, why did it change from `https` to `http`?

Reply:
you're right, I don't know why. I can revert this change anyway.